app/components/shop-sidebar/categories/subcategories.tsx

```
@@ -0,0 +1,56 @@
+      <ul className="flex flex-col gap-2">
+        <li className="bg-gray-200 px-2 py-1 mt-3 capitalize cursor-pointer text-sm">
+          {category || subcategory}
+        </li>
+        {subCategories &&
+          subCategories.map((subcategory) => (
+            <SubcategoryLink
+              key={subcategory.subcategory_id}
+              subcategoryName={subcategory.subcategory_name}
+              subcategoryId={subcategory.subcategory_id}
+            />
+          ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)